onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -362,8 +362,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "past state for key and value with shape (2, batch_size, num_heads, past_sequence_length, head_size)"
                "When past_present_share_buffer is set, "
                "its shape is (2, batch_size, num_heads, max_sequence_length, head_size)",
```

Comment:
I saw the cached past is reordered to [B, N, head_size / x, max_length, x], do we still accept the (2, batch_size, num_heads, past_sequence_length, head_size) here?

Reply:
We accept (2, batch_size, num_heads, past_sequence_length, head_size) as the shape of the tensor. Internally, the buffer of the key is reordered to [B, N, head_size / x, max_length, x]. I can include a line in the docs about this.

Reply:
Included some lines about keys buffer re-ordering in the docs